capstone/capapi/serializers.py

```diff
@@ -6,34 +6,50 @@
 from capdb import models
 from .models import APIUser
 from .resources import email
+from .permissions import get_single_casebody_permissions
+from scripts.generate_case_html import generate_html
+from scripts import helpers
 
 logger = logging.getLogger(__name__)
 
 
 class CitationSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Citation
-        fields = ('url', 'type', 'cite')
+        fields = ('type', 'cite', 'normalized_cite')
+
+
+class CitationWithCaseSerializer(CitationSerializer):
+    case_url = serializers.HyperlinkedRelatedField(source='case', view_name='casemetadata-detail',
+                                                   read_only=True, lookup_field='id')
+    class Meta:
+        model = models.Citation
+        fields = CitationSerializer.Meta.fields + ('case_id', 'case_url')
 
 
 class CaseSerializer(serializers.HyperlinkedModelSerializer):
     url = serializers.HyperlinkedIdentityField(
-        view_name="casemetadata-detail",
-        lookup_field='slug')
+        view_name="casemetadata-detail", lookup_field="id")
     jurisdiction = serializers.ReadOnlyField(source='jurisdiction.name')
     jurisdiction_url = serializers.HyperlinkedRelatedField(source='jurisdiction', view_name='jurisdiction-detail', read_only=True, lookup_field='slug')
     court = serializers.ReadOnlyField(source='court.name')
     court_url = serializers.HyperlinkedRelatedField(source='court', view_name='court-detail', read_only=True, lookup_field='slug')
     reporter = serializers.ReadOnlyField(source='reporter.full_name')
     reporter_url = serializers.HyperlinkedRelatedField(source='reporter', view_name='reporter-detail', read_only=True)
-    citations = CitationSerializer(many=True)
+    citations = serializers.SerializerMethodField()
     volume = serializers.ReadOnlyField(source='volume.volume_number')
     volume_url = serializers.HyperlinkedRelatedField(source='volume', view_name='volumemetadata-detail', read_only=True)
 
+    @staticmethod
```

Comment:
woops, no! I kept sticking to fields instead of going with nested relationships because of the need to obscure what I'd guess is a bunch of unnecessary data in the case list view. But here, we're exposing everything we have to offer for citations anyway, so might as well go with the citation serializer instead.
I will keep the rest of the fields (minus jurisdiction) as fields though — otherwise it gets overly cluttered with information